iteration/iteration.go

```diff
@@ -58,13 +62,41 @@ type GormIterationRepository struct {
 	db *gorm.DB
 }
 
+// ConvertToLtreeFormat returns LTREE valid string
+func ConvertToLtreeFormat(uuid string) string {
+	//Ltree allows only "_" as a special character.
+	return strings.Replace(uuid, "-", "_", -1)
+}
+
+// ConvertFromLtreeFormat returns UUID in form of string
+func ConvertFromLtreeFormat(uuid string) string {
+	// Ltree allows only "_" as a special character.
+	converted := strings.Replace(uuid, "_", "-", -1)
+	converted = strings.Replace(converted, PathSepInDatabase, PathSepInService, -1)
+	return converted
+}
+
+// LoadMultiple returns multiple instances of iteration.Iteration
+func (m *GormIterationRepository) LoadMultiple(ctx context.Context, ids []uuid.UUID) ([]*Iteration, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "Area", "getmultiple"}, time.Now())
```

Comment:
my bad, copy and paste and `mistake` !
corrected in https://github.com/almighty/almighty-core/pull/713/commits/90e0bf846968b5afcfbe3546ed58337c373581b8